lang/java/maven-plugin/src/test/resources/unit/idl/pom-injecting-velocity-tools.xml

```diff
@@ -22,7 +22,7 @@
   <parent>
     <artifactId>avro-parent</artifactId>
     <groupId>org.apache.avro</groupId>
-    <version>1.11.0-SNAPSHOT</version>
+    <version>1.12.0-SNAPSHOT</version>
```

Comment:
It seems something is wrong with the release process. Several pom.xml files were still pointing to the old 1.11.0-SNAPSHOT parent.

Reply:
Good catch, this should have been done in https://cwiki.apache.org/confluence/display/AVRO/How+To+Release#HowToRelease-Branching (Step 2c)!